src/host.rs

```diff
@@ -297,6 +299,70 @@ impl Drop for PluginInstance {
     }
 }
 
+/// The editor of an externally loaded VST plugin.
+pub struct PluginEditor {
```

Comment:
This struct should not be public. `PluginInstance` provides a `Box<dyn Editor>` via `get_editor`, but the concrete type is an implementation detail. Also, for consistency with the other types in the file, I suggest it is named `EditorInstance`.

Reply:
Done, thanks :)